lib/rex/proto/mssql/client_mixin.rb

```diff
@@ -178,6 +178,13 @@ def mssql_parse_tds_reply(data, info)
         col[:id] = :int
         col[:int_size] = data.slice!(0, 1).unpack('C')[0]
 
+      when 50
+        col[:id] = :sybbit
```

Comment:
Where does `sybbit` come from? 👀 

Is that this? 

>  BITTYPE          =   %x32  ; Bit 
> https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/859eb3d2-80d3-40f6-a637-414552c9c552

Reply:
Oh, is the existing implementation and terminology from freetds spec?

```
0x32	50	SYBBIT		no	0

...

0x68	104	SYBBITN		yes	1

https://www.freetds.org/tds.html
```

Reply:
I got bitntype from the microsoft tds spec and sybbit from the freetds spec - I'm open to feedback on the names. I think it would be reasonable to change to both of the freetds names, or :bit and :nullable_bit to be more specific?

Reply:
Ah; From what I understand, in freetds - `syb` is just a prefix that C developers use, and it comes from the `sybase` product as far as I can see:

TDS Version | Supported Products
-- | --
4.2 | Sybase SQL Server < 10 and Microsoft SQL Server 6.5
5.0 | Sybase SQL Server >= 10
7.0 | Microsoft SQL Server 7.0
7.1 | Microsoft SQL Server 2000
7.2 | Microsoft SQL Server 2005

So the naming convention should just probably be:

```
SYBBIT => bit
SYBBITN => bitn
```